src/website/app/demos/Menu/bestPractices.js

```diff
@@ -33,55 +33,55 @@ export default {
   menu: [
     {
       type: 'do',
-      title: 'use the appropriate variant for your intent',
+      backgroundColor,
+      description:
+        'Use the [appropriate variant](/color/#guidelines-variants) for your intent.',
       example: (
         <DemoLayout>
           <Menu>
             <MenuItem>Share</MenuItem>
             <MenuItem>Edit</MenuItem>
             <MenuItem>Duplicate</MenuItem>
-            <MenuItem variant="danger">Delete</MenuItem>
+            <MenuItem variant="success">Save</MenuItem>
           </Menu>
         </DemoLayout>
-      ),
-      description: `Match Mineral UI's [pre-defined variants](/color/#guidelines-variants) with semantic purpose.`,
-      backgroundColor
+      )
     },
     {
-      type: 'do',
-      title: 'use clear labeling for option messaging',
+      type: 'dont',
+      backgroundColor,
+      description:
+        "Don't use a variant that differs from intent, as this will confusion.",
```

Comment:
this sentence is missing the word, "because". I think it should also be moved to the MenuItem doc